src/extension.ts

```diff
@@ -21,6 +22,8 @@ export async function activate(context: vscode.ExtensionContext) {
 
   tailscaleInstance = await Tailscale.withInit(vscode);
 
+  const configManager = ConfigManager.withContext(context);
```

Comment:
I was surprised to see the config manager taking the entire Extension Context. Turns out it only needs it for the global storage path. 

Can we have the ConfigManager just take that string as an argument instead of the entire context? This way we know exactly what it's being used for without having to dig through the code.

Reply:
Yup, done.